firmware/ec/src/filesystem/fs_wrapper.c

```diff
@@ -39,9 +39,6 @@
 #define READ_SIZE 256
 #define WRITE_SIZE 256
 
-static Queue_Struct fsRxMsg;
```

Comment:
It's presently unused, which is why we get the warning.  Should we use a pragma around it?

Reply:
i had a discussion with the team here , we decided to go ahead with your changes of removing the these unused variables . This might create merge conflicts when alert implementation  is getting merged , we will take care of it then .